CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+
+Be especially careful of “free textures” (or “free sounds” or “free graphics”) sites. Although these would appear at first to be okay to use, many are free for “non-commercial use only.” One of the things we want to be able to do is put this in GNU/Linux distributions (which can be sold or developed commercially).
+
+## Levels
+
+Not all levels for LibreQuake must be vanilla-compatible Winquake/vanilla-compatible levels will have their own episode
+
+It is sensible to also heed the following guidelines:
+
+1. Make sure that skill levels are implemented, and that all multiplayer start points, both cooperative and deathmatch, are present.
+
+2. Try to make levels appropriately difficult for their position within the progression of the game. Also bear in mind that not all players may be as skilled a player as you.
+
+3. While unrestricted by limits, do not make excessively complicated scenes. It is desirable that LibreQuake levels should be playable on low-powered hardware, such as phones and old computers.
+
+
+## Branches
+
+Please do not generalize branches. Use branches for specific features or concepts.
+
+If you are a Collaborator (meaning, if you are added to the LibreQuake repository and have administrative control) it is required that your branch naming scheme is as follows:
+
+`{username}/{change-subject}`
+
+Shorthand is allowed for `username`. For instance, if your GitHub username is `lavenderdotpet`, using `lav` would be appropriate. However, these must be easily connected to your GitHub username to track who owns a branch.
+
+The `change-subject` field should be a **very** brief issue or feature or similar concept. 
+
+An example branch name that fits this standard would be `cypress/quakec-stringopts` for a branch owned by CypressImplex that makes QuakeC related string optimizations.
+
+External contributors (non-Collaborators) are not expected to follow this name convention, but should still refrain from using generalized branch names.
+
+## Commits
+
+In general, this guide is a must-read when writing good commit messages to begin with: https://cbea.ms/git-commit/
+
+Additionally, as this repository contains multiple diverse components, we ask that you prefix your commit message pertaining to the component you are modifying. Here are acceptable prefixes to use:
+
+* `BUILD`: Modification of our build scripts and/or tools.
+* `CI`: Modification of the GitHub Actions Pipeline(s).
+* `GFX`: Modification of **game** (not map) textures.
+* `GH`: Modifiation of any repository-related elements (screenshots, readme, etc.)
+* `MAPS`: Modification of `.map` files.
+* `MODELS`: Modification of models, both `.blend` and exported `.mdl`
+* `QC`: Modification of QuakeC source code.
+* `AUDIO`: Modification of game sound effects or music.
+* `WADS`: Modification of **map** textures, to be packed into a `.wad` during the build process.
```

Comment:
I would like to have this list accessible when creating new PR.

Reply:
also same would want this 
can hide it within this
```<!-- This content will not appear in the rendered Markdown -->```